views/jobs.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+
+<head>
+    <title>Job Roles</title>
+
+    <style>
+        body {
+            padding: 50px;
+            font: 14px "Lucida Grande", Helvetica, Arial, sans-serif;
+        }
+
+        a {
+            color: #00B7FF;
+        }
+
+        ul {
+            list-style-type: none;
+            margin: 0;
+            padding: 0;
+            overflow: hidden;
+            background-color: lightskyblue;
+        }
+
+        li {
+            float: left;
+        }
+
+        li a {
+            display: block;
+            color: black;
+            text-align: center;
+            padding: 14px 16px;
+            text-decoration: none;
+        }
+
+        /* Change the link color to #111 (black) on hover */
+        li a:hover {
+            background-color: white;
+        }
+/* 
+        #logo {
+            
+            width: 50px;
+            height: 100px;
+            top: 0;
+            left: 0;
+        } */
+        img{
+            width: 470px;
+            height: 200px;
+        } 
+        table{
+            border: 1px solid;
+            width: 1000px;
+            border-collapse: collapse;
+            text-align:center;
+            padding:10px;
+        }
+        th{
+            padding:10px;
+            width: 100px;
+            background-color: #00B7FF;
+        }
+        td{
+            padding-bottom: 1em;
+            width: 100px;
+        }
+    </style>
+
+
+</head>
+
+<body>
+
+    <div id="logo">
+        <img  src="https://upload.wikimedia.org/wikipedia/en/3/3e/Kainos_logo.jpg">
+    </div>
+
+    <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="http://localhost:3000/job-roles">Job Roles</a></li>
+        <li><a href="#">Capabilities</a></li>
+    </ul>
+
+
+    <br><br>
+
+
+    {% block main %}
+    <table>
+        {% if jobRoles and jobRoles.length > 0 %}
```

Comment:
What does the page display if there are no job roles?

Reply:
Message was thumbed up but don't see any changes or a response? :)

Reply:
Hey, sorry for the lack of a response - the jobcontroller renders error.html in such a scenario, using an appropriate error message (passed through as an errormessage variable to nunjucks)